### ISSUE TITLE
i cannot select the TTS from google

### ISSUE THREAD

**Issue by MJ933** [LNReader/lnreader#1190]
### Steps to reproduce

before i click on TTS voice i only can see "system (system) " but when i click on it it show me a search bar and nothing more

### Expected behavior

it should show TTS from Google and from Hawaii 

### Actual behavior

it not showing any thing but the search bar 

### LNReader version

2.0.0 

### Android version

android 10

### Device

 Hawaii y9 prime 

### Other details

i also has too many problems in this beta especially in my Hawaii y9 prime in design and also in too many things but these other problems is not appearing in my Xiaomi pad 5  and the app becomes very slow when i open novel from any plugin and over all the app is slower then the old version of 1.1.19
thank you for your time.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with a source, I should be opening an issue in the [sources repository](https://github.com/lnreader/lnreader-sources/issues/new/choose).
- [X] I have updated the app to version **[2.0.0](https://github.com/lnreader/lnreader/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by Palloxin** [comment]
I agree on slower opening of entries(novel screen). But where else is it slower?

**Comment by MJ933** [comment]
Now I did this 
![Screenshot_20240808_130026_com.android.settings.jpg](https://github.com/user-attachments/assets/36dbf8ad-0833-48e9-bdc6-6b58b1befa9e)

Now it has the correct voice but it not has the correct speed it should be 2x faster why it speed looks like at 1x?

**Comment by nyagami** [comment]
You can change rate and pitch in settings in next release

**Comment by Anan5a** [comment]
I have the same issue on `realme x2` Selecting google from tts settings does not use the google tts on the app, but in the settings app the sample is spoken in correct voice, and system wide tts (speak aloud) use the correct google voice. @nyagami what can i do?

**Comment by nyagami** [comment]
> I have the same issue on `realme x2` Selecting google from tts settings does not use the google tts on the app, but in the settings app the sample is spoken in correct voice, and system wide tts (speak aloud) use the correct google voice. @nyagami what can i do?

I have no idea for that, try using the latest build app from github action
